cypress/e2e/googlemap.cy.ts

```diff
@@ -0,0 +1,29 @@
+describe('Google Maps API', ()=>{
+    beforeEach(()=>{
+        cy.intercept('GET', '**/autocomplete/**', (req)=>{
+            if(req.url.includes('london')){
+                req.reply([
+                    {
+                        description:'London, UK',
+                        place_id: 'some_place_id'
+                    }
+                ])
+            }
+        })
+    })
+
+    it('correctly locates center', ()=>{
+        cy.visit('/')
+        cy.get('#monocle').click()
+        .wait(2000)
```

Comment:
it would be nice to avoid this wait()/ its definitely bad practice.

Reply:
if your waiting for a 'side effect' there is a specific wait() that awaits a request to resolve rather than waiting a  fixed time.

Reply:
Can you point me to an article where it says it's bad practice to use the wait in this way? would like to read up on it